### DISCUSSION TITLE
How to install apps using user context?

### DISCUSSION THREAD

**Discussion by atp0303** [svrooij/WinTuner#78] [General]
After uploading the option to set the install behavior to user is greyed out and default to system.

We have some apps which is required to run in the user context. Is there an arg we can use to configure this setting?

![image](https://github.com/svrooij/WingetIntune/assets/1209692/4ee6dbf0-b237-41df-926c-bf65108608d1)

**Comment by DKhalaf** [comment]
Interesting. The apps i'm publishing are set as User by default and I cannot switch that around to System with --installer-context System.

**Comment by svrooij** [comment]
This piece of software depends upon `winget` to get it's packages. And it tries to match the scope specified in the manifest, so [Azure CLI](https://github.com/microsoft/winget-pkgs/blob/d5c88ebeb1f1721b627387965cd3a2300e750b5c/manifests/m/Microsoft/AzureCLI/2.59.0/Microsoft.AzureCLI.installer.yaml#L8) has `Scope: machine`, it will get installed as System. [BingWallpaper](https://github.com/microsoft/winget-pkgs/blob/d5c88ebeb1f1721b627387965cd3a2300e750b5c/manifests/m/Microsoft/BingWallpaper/2.0.0.1/Microsoft.BingWallpaper.installer.yaml) uses an older version manifest file, which does not require specifying a scope (system is assumed).

For instance [OhMyPosh](https://github.com/microsoft/winget-pkgs/blob/d5c88ebeb1f1721b627387965cd3a2300e750b5c/manifests/j/JanDeDobbeleer/OhMyPosh/22.0.3/JanDeDobbeleer.OhMyPosh.installer.yaml#L22) has the scope defined per installer. Which allows you to pick either `system` or `user`.

I was not aware of apps that did not have a scope defined in the manifest. What is the best solution to fix this?
